src/main/java/net/snowflake/client/jdbc/cloud/storage/SnowflakeAzureClient.java

```diff
@@ -51,7 +62,7 @@
   private final static SFLogger logger =
       SFLoggerFactory.getLogger(SnowflakeAzureClient.class);
   private OperationContext opContext = null;
-  private static SFSession session;
```

Comment:
oh, this looks very interesting fix.

Reply:
figured out how to change to non-static! Added session parameter to setupAzureClient() instead of initializing in constructor.